API/Intro.md

```diff
@@ -0,0 +1,151 @@
+APIs are sets of defined protocols enabling applications to communicate with other systems, that can also be applications. In the context of the Web, APIs are a way for Web clients (browsers, mobile apps, terminals, servers, and other platforms) to communicate with Web servers.
+
+Our APIs use the HTTP protocol and are usually in JSON format, but if necessary, we might implement file uploads, XML or something else the project requires.
+
+The following chapters provide information on our practices regarding JSON APIs. Although some gems are mentioned throughout, primary focus is on aspects surrounding API design and delivery.
+
+## API design principles
+
+Before talking about specific topics like authentication and pagination, we should outline a set of principles all APIs should follow. Since it's challenging to understand generic definitions, all of the principles will be introduced with an example showing what happens when they're *not* followed to try to illustrate the importance of following them.
+
+*Note*: all of the examples below are inspired by situations developers have been in when dealing with terrible APIs.
```

Comment:
```suggestion
*Note*: all the examples below are inspired by situations developers have found themselves in when dealing with terrible APIs.
```

Reply:
thanks! One more [grammar rule](https://www.grammar-monster.com/lessons/all_of.htm) to remember :D